### ISSUE TITLE
Overview in each document

### ISSUE THREAD

**Issue by hnsengimana** [kevinaangstadt/CS340-S23-Notes#133]
I've noticed that many of the lecture notes do not currently have an overview section at the beginning. Not only to follow the style guide but to improve the usability of these notes we should include overviews.

**Comment by AidanMarler123** [comment]
I agree, I think if we all create overview sections that reviewing and understanding the content will be much easier.

**Comment by jabelm20** [comment]
I like this idea. I'll add this to mine now!
